src/forms/textarea.jsx

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import classnames from 'classnames';
 import Input from './input';
 import PropTypes from 'prop-types';
 
 class TextArea extends Input {
 
   render() {
-    const {label, hint, error, name, rows, cols, disabled, readonly, ...other} = this.props;
+    const {label, hint, error, name, rows, cols, disabled, readonly, className, ...other} = this.props;
 
     return <div className={this.errorClass('govuk-form-group')}>
       <label className="govuk-label" htmlFor={this.id()}>{label}</label>
       { hint && <span id={this.id() + '-hint'} className="govuk-hint">{hint}</span> }
       { error && <span id={this.id() + '-error'} className="govuk-error-message">{error}</span> }
       <textarea
-        className={this.errorClass('govuk-textarea')}
+        className={classnames(this.errorClass('govuk-textarea'), className)}
```

Comment:
When we did a similar thing with radio buttons the classname was applied to the wrapper element rather than the inputs themselves, can we do similar here too?

Reply:
yes, good spot